client/app/components/packages/pas-form/edit.js

```diff
@@ -64,19 +68,29 @@ export default class PasFormComponent extends Component {
     return this.submittableChanges.isValid;
   }
 
-  @action
-  async save() {
-    await this.saveableChanges.save();
-
-    this.args.onSave();
+  get isSavingOrSubmitting() {
+    return this.save.isRunning || this.submit.isRunning;
   }
```

Comment:
Wondering about the naming here... I wonder if `isSettling` is a better term?

Reply:
Originally thought a more descriptive name would be clearer than something like isProcessing. 

Maybe isPatching would imply the database record is being modified either through a save or submit?

Reply:
Yeah, Idk, this is a bit nit from me so disregard. "patching" is very HTTP-specific, i just wonder if we can find a better than for what this is rather than a camel-case version describing the code itself ;P